File: src/main/java/net/tecdroid/constants/UnitConstants.java
package net.tecdroid.constants;

import edu.wpi.first.units.Units;
import edu.wpi.first.units.measure.Angle;

public class UnitConstants {
    public static final Angle FULL_ROTATION    = Units.Degrees.of(360);
    public static final Angle HALF_ROTATION    = FULL_ROTATION.div(2);
    public static final Angle QUARTER_ROTATION = HALF_ROTATION.div(2);
}

File: src/main/java/net/tecdroid/constants/UnitConstants.kt
package net.tecdroid.constants

import edu.wpi.first.units.Units.Rotations
import edu.wpi.first.units.measure.Angle

object UnitConstants {
    val fullRotation: Angle = Rotations.of(1.0)
    val halfRotation: Angle = fullRotation.div(2.0)
}
